### PR TITLE
Seblatre/scmtag

### LINKED ISSUE #11: Add a rule on scm > tag element in the pom.xml (maven-release plugin)
When using maven-release plugin, the goals release;branch and release:prepare are both modifiying the `scm > tag` element in the pom.xml.
* maven:prepare set `scm > tag` to the new Git branch 
maven-branch
* maven:prepare set `scm > tag` to the Git tag generated during the release process then put it back to head in the next commit for development
maven-release

Therefore, there can be conflicts on this item when merging pom.xml. A new rule could be added to manage such behavior automatically.

### PR DESCRIPTION
**Pull request by seblatre**
Fix #11 .

### CODE REVIEW

File: src/test/resources/rulesets/rulesetScmTag/ourStrategy/our.pom.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <dependencies>
+  	<dependency>
+  		<groupId>org.codehaus.mojo</groupId>
+  		<artifactId>versions-maven-plugin</artifactId>
+  		<version>2.1</version>
+  		<type>maven-plugin</type>
+  	</dependency>
+  </dependencies>
+  <scm>
+    <tag>base</tag>
```
Comment (cecom) [inline src/test/resources/rulesets/rulesetScmTag/ourStrategy/our.pom.xml]:
```suggestion
    <tag>our</tag>
```

File: src/test/resources/rulesets/rulesetScmTag/ourStrategy/their.pom.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <dependencies>
+  	<dependency>
+  		<groupId>org.codehaus.mojo</groupId>
+  		<artifactId>versions-maven-plugin</artifactId>
+  		<version>2.5</version>
+  		<type>maven-plugin</type>
+  	</dependency>
+  </dependencies>
+  <scm>
+    <tag>base</tag>
```
Comment (cecom) [inline src/test/resources/rulesets/rulesetScmTag/ourStrategy/their.pom.xml]:
```suggestion
    <tag>their</tag>
```

File: src/test/resources/rulesets/rulesetScmTag/theirStrategy/our.pom.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  <dependencies>
+  	<dependency>
+  		<groupId>org.codehaus.mojo</groupId>
+  		<artifactId>versions-maven-plugin</artifactId>
+  		<version>2.1</version>
+  		<type>maven-plugin</type>
+  	</dependency>
+  </dependencies>
+  <scm>
+    <tag>base</tag>
```
Comment (cecom) [inline src/test/resources/rulesets/rulesetScmTag/theirStrategy/our.pom.xml]:
```suggestion
    <tag>our</tag>
```

File: src/test/resources/rulesets/rulesetScmTag/theirStrategy/their.pom.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  <dependencies>
+  	<dependency>
+  		<groupId>org.codehaus.mojo</groupId>
+  		<artifactId>versions-maven-plugin</artifactId>
+  		<version>2.5</version>
+  		<type>maven-plugin</type>
+  	</dependency>
+  </dependencies>
+  <scm>
+    <tag>base</tag>
```
Comment (cecom) [inline src/test/resources/rulesets/rulesetScmTag/theirStrategy/their.pom.xml]:
```suggestion
    <tag>their</tag>
```

**Review by cecom — CHANGES_REQUESTED** [review]
And plz rebase on develop branch and choose as target develop in the pull request.

Otherwise it looks good 👍

**Comment by seblatre** [comment]
@cecom, sorry I'm new to github, I'm learning ^^
Hope this is right now.

**Review by cecom — COMMENTED** [review]
looks good

**Pull request merged**